velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -887,17 +887,17 @@ TEST_F(DateTimeFunctionsTest, fromUnixtime) {
       fromUnixTime(getUnixTime("2020-06-30 23:59:59"), "yyyy-MM-dd HH:mm:ss"),
       "2020-06-30 23:59:59");
 
+  // 8 hours ahead UTC.
+  setQueryTimeZone("Asia/Shanghai");
```

Comment:
Do we need this change to reproduce this issue?

Reply:
Yes, we can't reproduce this issue without setting the timezone.

Reply:
Yes, Spark allows overflow and is aligned with current implementation in Velox. The result for fromUnixTime(INT64_MAX, "yyyy-MM-dd HH:mm:ss") without setting timezone is "1969-12-31 23:59:59" Here is a test :
```
spark-sql> select from_unixtime(9223372036854775807);
from_unixtime(9223372036854775807, yyyy-MM-dd HH:mm:ss)
1969-12-31 23:59:59
```

Reply:
Thanks for checking.